### ISSUE TITLE
Unable to publish with customeized attributes 

### ISSUE THREAD

**Issue by ivanchy** [open-webrtc-toolkit/owt-server#1248]
I have set an attributes in LocalStream，but after publication the RemoteStream attributes become undefined.Can i do this with any methods?

**Comment by starwarfan** [comment]
which version of code did you use?

**Comment by ivanchy** [comment]
> which version of code did you use?

The latest version 5.0，at the below: 

https://github.com/open-webrtc-toolkit/owt-server/blob/master/test/testingpage/script2.js

Line 1822 ，  { testAttributes: 'test' }

**Comment by starwarfan** [comment]
In latest 5.0.x branch, you should be able to get it through `RemoteStream.attributes`.

**Comment by ivanchy** [comment]
> In latest 5.0.x branch, you should be able to get it through `RemoteStream.attributes`.

I have tried in the 5.0 version. 

In the console logging, LocalStream can display the attributes
![圖片](https://user-images.githubusercontent.com/16917916/190342416-56955fa0-2077-442d-bfee-6b57cfe20b9c.png)

However, after publication RemoteStream.attributes become undefined
![圖片](https://user-images.githubusercontent.com/16917916/190342701-24078988-af8a-45f1-b3d3-c3ac8cd6509c.png)

Is it a bug? or I should try the 5.0.1 version?

**Comment by starwarfan** [comment]
Yes. You need try 5.0.1 version.

**Comment by Phillipip** [comment]
Is it possible to change these attributes later?

**Comment by starwarfan** [comment]
> Is it possible to change these attributes later?

No. The attributes are decided when published.

**Comment by Phillipip** [comment]
Is it also not possible to change it via the API?

**Comment by starwarfan** [comment]
> Is it also not possible to change it via the API?

No. You may manually save it in one place and notify others about its changes.
